sdf.md

```diff
@@ -263,29 +298,46 @@ Class:
 : Abstract term for the information that is contained in groups
   identified by a Class Name Keyword.
 
-Property:
-: An affordance that can potentially be used to read, write, and/or
-  observe state (current/stored information) on a Grouping.
-  (Note that Entries are often called
-  properties in other environments; in this document, the term
-  Property is specifically reserved for affordances, even if the map
-  key "properties" might be imported from a data definition language
-  with the other semantics.)
+Quality:
+: A metadata item in a definition or declaration which says something
+  about that definition or declaration.  A quality is represented in
+  SDF as an entry in a JSON map (JSON object) that serves as a definition
+  or declaration.
+  (The term "Quality" is used because another popular term,
+  "Property", already has a different meaning.)
 
-Action:
-: An affordance that can potentially be used to perform a named operation on a Grouping.
+Definition:
+: An entry in a Definition Group.
+  The entry creates a new semantic term for use in SDF models and
+  associates it with a set of qualities.
+  Unless the Class Name Keyword of the Group also makes it a
+  Declaration (see {{definitions-block}}), a definition just defines a
+  term, it does not create a component item within the enclosing
+  definition.
 
-Event:
-: An affordance that can potentially be used to obtain information about what happened to a Grouping.
+Declaration:
+: A definition within an enclosing
+  definition that is intended to create a component item within that
+  enclosing definition.  Every declaration can also be used as a
+  definition for reference elsewhere.
+
+Grouping:
+: An sdfThing or sdfObject, i.e., (directly or indirectly) a
```

Comment:
Should these also use backticks? 

```suggestion
: An `sdfThing` or `sdfObject`, i.e., (directly or indirectly) a
```

(Although this is very annoying and probably only relevant for the HTML-rendered version, I guess there might be one final round needed, where this is adjusted across the document, as I think the usage of backticks is not fully consistent at the moment.)